docs/CONTRIBUTING.md

```diff
@@ -2,46 +2,44 @@
 
 We're really glad you're reading this, you might be interested by the code source of the Decidim app by Open Source Politics decidim application.
 
-If you haven't already, come find the Decidim community in [Gitter](https://gitter.im/decidim/decidim). Community wants you working on things you're excited about.
+If you haven't already, come find the Decidim community in [Matrix](https://app.element.io/#/room/#decidim:matrix.org). Community wants you working on things you're excited about 🔥.
 
 ## Did you find a bug?
 
-* **Please, do not open up a GitHub issue if the bug is a security vulnerability**, and instead follow instructions in the dedicated [security guide](./docs/SECURITY.md).
+* **Please, do not open up a GitHub issue if the bug is a security vulnerability**, and instead follow instructions in the dedicated [security guide](./SECURITY.md).
 
-* **Ensure the bug was not already reported** by searching on GitHub under [Issues](https://github.com/decidim/decidim/issues) and on [Metadecidim](https://meta.decidim.org/processes/bug-report/f/210/proposals).
+* **Ensure the bug was not already reported** by searching on GitHub under [Issues](https://github.com/OpenSourcePolitics/decidim-app/issues).
 
-* If you're unable to find an open issue addressing the problem, [open a new one on the current repository](https://github.com/OpenSourcePolitics/decidim-app). Be sure to include a **title and clear description**, as much relevant information as possible, and a **code sample** or an **executable test case** demonstrating the expected behavior that is not occurring.
+* If you're unable to find an open issue addressing the problem, [open a new one on the current repository](https://github.com/OpenSourcePolitics/decidim-app).
 
 ## Do you want to contribute on a patch or a feature ?
-
-* Fork this repository
+* [Fork this repository](https://github.com/OpenSourcePolitics/decidim-app/fork) 
 * Create a new branch on your fork
-* Once your PR is ready, open a draft PR on the current repository
+* Once your PR is ready, open a draft PR on our repository
 * Once all checks are green (GG!), please mark your PR as "Ready for review".
-
 * Ensure the PR description clearly describes the problem and solution. Include the relevant issue number if applicable.
 
 ## Do you intend to add a new feature or change an existing one?
-
-* Open an issue on the current repository and describe it clearly
+*  🇫🇷 Please create a new feature proposal on our french open roadmap [here](https://club.decidim.opensourcepolitics.eu/assemblies/feuille-de-route/f/232/)
+* 🌎 Open an issue on the current repository and describe it following our template
```

Comment:
```suggestion
* 🌎 Create a proposal on the [Meta.decidim platform](https://meta.decidim.org/processes/roadmap/f/122/), to see if it can be accepted in the core of the software
```